src/approaches/reinforcement_learning_approach.py

```diff
@@ -0,0 +1,109 @@
+"""A bilevel planning approach that learns NSRTs from an offline dataset, and
+continues learning options through reinforcement learning."""
+
+from typing import Callable, List, Optional, Sequence, Set
+
+import dill as pkl
+from gym.spaces import Box
+
+from predicators.src import utils
+from predicators.src.approaches.base_approach import ApproachFailure, \
+    ApproachTimeout
+from predicators.src.approaches.nsrt_learning_approach import \
+    NSRTLearningApproach
+from predicators.src.nsrt_learning.nsrt_learning_main import \
+    learn_pruned_nsrts_from_data
+from predicators.src.settings import CFG
+from predicators.src.structs import NSRT, Dataset, GroundAtom, \
+    InteractionRequest, InteractionResult, LowLevelTrajectory, \
+    ParameterizedOption, Predicate, State, Task, Type
+
+
+class ReinforcementLearningApproach(NSRTLearningApproach):
+    """A bilevel planning approach that learns NSRTs."""
+
+    def __init__(self, initial_predicates: Set[Predicate],
+                 initial_options: Set[ParameterizedOption], types: Set[Type],
+                 action_space: Box, train_tasks: List[Task]) -> None:
+        super().__init__(initial_predicates, initial_options, types,
+                         action_space, train_tasks)
+        self._nsrts: Set[NSRT] = set()
+        self.online_learning_cycle = 0
+        self.initial_dataset: List[LowLevelTrajectory] = []
+        self.online_dataset: List[LowLevelTrajectory] = []
+        # initialize option learner?
+
+    def _learn_pruned_nsrts(self,
+                            initial_trajectories: List[LowLevelTrajectory],
+                            trajectories: List[LowLevelTrajectory],
+                            online_learning_cycle: Optional[int]) -> None:
+        self._nsrts = learn_pruned_nsrts_from_data(
+            initial_trajectories,
+            trajectories,
+            self._train_tasks,
+            self._get_current_predicates(),
+            self._action_space,
+            sampler_learner=CFG.sampler_learner)
+        save_path = utils.get_approach_save_path_str()
+        with open(f"{save_path}_{online_learning_cycle}.NSRTs", "wb") as f:
+            pkl.dump(self._nsrts, f)
+
+    @classmethod
+    def get_name(cls) -> str:
+        return "reinforcement_learning"
+
+    def learn_from_offline_dataset(self, dataset: Dataset) -> None:
+        # The only thing we need to do here is learn NSRTs,
```

Comment:
remove this copied comment, and rewrite to something like:
```
self.initial_dataset = dataset.trajectories
super().learn_from_offline_dataset(dataset)
```
with the key thing being calling the parent class method instead of copying it.